c++/kj-async.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@prefix@
+libdir=@libdir@
+includedir=@includedir@
+
+Name: Cap'n Proto
```

Comment:
Probably the name should be "KJ Async Framework Library", and for kj.pc "KJ Framework Library".